game_patch/os/commands.cpp

```diff
@@ -104,6 +107,59 @@ FunHook<void()> verify_level_cmd_hook{
     }
 };
 
+void handle_camera_command(FunHook<void()>& hook)
+{
+    if (!(rf::level.flags & rf::LEVEL_LOADED)) {
+        rf::console::print("No level loaded!");
+        return;
+    }
+
+    if (rf::is_multi) {
+        rf::console::print("That command can't be used in multiplayer.");
+        return;
+    }
+
+    hook.call_target();
+
+    const rf::CameraMode current_mode = rf::camera_get_mode(*rf::local_player->cam);
+
+    std::string mode_text = (current_mode == rf::CAMERA_FIRST_PERSON)   ? "first person"
+                            : (current_mode == rf::CAMERA_THIRD_PERSON) ? "third person"
+                            : (current_mode == rf::CAMERA_FREELOOK)     ? "free look"
+                                                                        : "unknown";
+
+    std::string helper_text =
+        (current_mode == rf::CAMERA_FIRST_PERSON) ? "" : " Use `camera1` to return to first person.";
+
+    rf::console::print("Camera mode set to {}.{}", mode_text, helper_text);    
```

Comment:
Too much ternary imo. Potential rewrite:
```suggestion
    std::string_view camera_mode_to_string_view(const rf::CameraMode mode) {
        switch (mode) {
            switch rf::CAMERA_FIRST_PERSON: return std::string_view{"first person"};
            switch rf::CAMERA_FREELOOK: return std::string_view{"free look"};
            switch rf::CAMERA_THIRD_PERSON: return std::string_view{"third person"};
            default: return std::string_view{"unknown camera mode"};
        }
    }

    const std::string_view mode_text = camera_mode_to_string_view(mode);
    if (mode == rf::CAMERA_FIRST_PERSON) {
        rf::console::print("Your camera mode was set to {}", mode_text);    
    } else {
        rf::console::print("Your camera mode was set to {}. Use `camera1` to return to first person", mode_text);    
    }
```